test/jdk/tools/jpackage/macosx/base/SigningBase.java

```diff
@@ -32,9 +32,23 @@
 
 public class SigningBase {
 
+    enum CertIndex {
+        ASCII_INDEX(0),
+        UNICODE_INDEX(0),
```

Comment:
This one should be 1. Looks like we had a bug.

Reply:
Fixed